easybuild/easyconfigs/e/ELSI-RCI/ELSI-RCI-0.1.0-foss-2021a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'ELSI-RCI'
+version = '0.1.0'
+
+homepage = 'https://wordpress.elsi-interchange.org/'
+description = """ELSI-RCI provides and enhances open-source software packages
+which iteratively solve or circumvent eigenvalue problems in self-consistent
+field calculations based on the Kohn-Sham density-functional theory.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'usempi': False, 'pic': True}
+
+source_urls = ['https://gitlab.com/elsi_project/elsi_rci/-/archive/%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['ELSI-RCI-0.1.0-cmake.patch']
+checksums = [
+    'a737673c1009b919be0daf86e86d1f4387a26711a10bda88a5f3f6c0b944c580',  # elsi-rci-0.1.0.tar.gz
+    'd8205d8f67775aa6cfc75813566a64b2a4b01c9b856268e457ef153b7988de82',  # ELSI-RCI-0.1.0-cmake.patch
+]
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+]
+
+
```

Comment:
Minor change here - can you remove the extra blank line.

Reply:
Done.